tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3288,6 +3292,11 @@ <h2>Show similarity to selected datapoint</h2>
             type: String,
           },
           maxInferenceEntriesPerRun: Number,
+          // Number of buckets when aggregating numeric features.
+          numBuckets: {
+            type: Number,
+            value: 10,
+          },
```

Comment:
can you add this as a numeric paper-input element below the slice by dropdown (when a numeric feature is selected) with a range of 2-10 and default value of 2? can use a <template is="dom-if...> wrapping the element so it only appears in that case. same thing with another dropdown under the secondary slice if that is numeric too. then ensure the buckets are recalculated whenever numPrimaryBuckets or numSecondaryBuckets changes.

Reply:
:+1:
![Screenshot from 2019-09-19 16-36-07](https://user-images.githubusercontent.com/6004563/65259134-162a5e80-dafc-11e9-981c-eebed2e4048d.png)